tools/docker/bin/run.sh

```diff
@@ -79,14 +79,15 @@ fi
 if [ "$COMPOSE_PROJECT_NAME" == "jetpack_dev" ] ; then
 	# If we don't have the wordpress test helpers, download them
 	if [ ! -d /tmp/wordpress-develop/tests ]; then
+		CUR_WP_VERSION=$(wp --allow-root core version);
```

Comment:
I've updated a few lines in this file, but I'm not sure when this runs or how to test it...

Reply:
This file runs as the default command whenever the container is started.

This particular code path will run when `/tmp/wordpress-develop/tests` (which is mounted from the host's `tools/docker/wordpress-develop`) doesn't exist.

Reply:
Note that `run.sh` didn't seem to run when the container started; I could only get it to run when the docker image was built.

Reply:
It does for me. 🤷